gbpservice/nfp/configurator/drivers/vpn/vyos/vyos_vpn_driver.py

```diff
@@ -0,0 +1,874 @@
+import copy
+import json
+import requests
+
+from gbpservice.nfp.configurator.agents import vpn
+from gbpservice.nfp.configurator.drivers.base import base_driver
+from gbpservice.nfp.configurator.lib import vpn_constants as const
+
+from oslo_concurrency import lockutils
+from oslo_config import cfg
+from oslo_log import log as logging
+
+LOG = logging.getLogger(__name__)
+
+auth_server_opts = [
```

Comment:
Where are these auth parameters used? I don't think we need any of these. Can we check?